### PR TITLE
Mention necessity to retain RSEs in conclusion

### PR DESCRIPTION
**Pull request by Bisaloo**
As discussed during the workshop in Paderborn

### CODE REVIEW

File: paper_teaching-learning-RSE.md [hunk 1]
```diff
@@ -712,6 +712,8 @@ Given the competencies and a demand(FIXME: Do the calculation) in the research l
 We closed with a discussion of possible structures and organization forms that educate new generations of RSEs in more structured programs than what is available today(FIXME: this is currently aspirational).
 Therefore this closes the gap, that the research landscape requires RSEs, but there are no structures where these persons are educated, by detailing the career path that a young person might want to take to become an RSE.(FIXME: also aspirational...)
 
+However, it is worth highlighting that answering the need for more RSEs can only be achieved by simultaneously having a clear training path, but also by ensuring that existing and trained RSEs are retained in academia. This paper focused on the first component: training RSEs, but we still believe that a discussion on incentives and brain drain by the private sector is necessary. This might be even more true for RSEs than traditional researchers as RSEs may have more opportunities to perform similar activities in the private sector, while benefiting from better salaries and working conditions.
```
Comment (knarrff) [inline paper_teaching-learning-RSE.md:715]: please put each sentence on a separate line

Comment (knarrff) [inline paper_teaching-learning-RSE.md:715]: "private sector" -> "commercial sector" ?

**Comment by mschwarzmeier** [comment]
I would add, that not only salaries and working conditions might be better in industry but in my mind even worse is that there seems no career path or even unlimited employment available to post-docs/experienced RSEs.

**Comment by CaptainSifff** [comment]
Yes, but the career path thing is also sth. that we have to solve in htis paper....

File: paper_teaching-learning-RSE.md (hunk 1 above)
Comment (CaptainSifff) [reply paper_teaching-learning-RSE.md]: fixed

**Pull request merged**